__tests__/UrqlClient_test.re

```diff
@@ -116,4 +116,4 @@ describe("UrqlClient", () => {
       )
     );
   });
-});
\ No newline at end of file
+});
```

Comment:
A _lot_ of these files just involved adding a carriage return to the file. I'm going to make sure that my local editor setup matches this project's `refmt` (I'm using `reason-language-server`, not sure if any of you happen to know if there's an easy way to check this).